src/Data/List/Relation/Unary/All/Properties.agda

```diff
@@ -33,15 +34,15 @@ open import Data.Maybe.Relation.Unary.Any as Maybe using (just)
 open import Data.Nat.Base using (zero; suc; s≤s; _<_; z<s; s<s)
 open import Data.Nat.Properties using (≤-refl; m≤n⇒m≤1+n)
 open import Data.Product.Base as Product using (_×_; _,_; uncurry; uncurry′)
-open import Function.Base
-open import Function.Bundles
+open import Function.Base using (_∘_; _$_; id; case_of_; flip)
+open import Function.Bundles using (_↠_; mk↠ₛ; _⇔_; mk⇔; _↔_; mk↔ₛ′; Equivalence)
 open import Level using (Level)
 open import Relation.Binary.Core using (REL)
 open import Relation.Binary.Bundles using (Setoid)
 import Relation.Binary.Definitions as B
-open import Relation.Binary.PropositionalEquality
+open import Relation.Binary.PropositionalEquality.Core
   using (_≡_; refl; cong; cong₂; _≗_)
-open import Relation.Nullary
+open import Relation.Nullary using (¬_; does; yes; no; _because_)
```

Comment:
These should be distributed over the three modules mentioned immediately below, and this import removed?
```agda
open import Relation.Nullary.Reflects using (invert)
open import Relation.Nullary.Negation using (¬_; contradiction)
open import Relation.Nullary.Decidable using (does; yes; no; _because_; ¬?; decidable-stable)```

Reply:
Could indeed be done (if it used `.Core` for two of them) but would have very little effect on the dependency graph, as `Relation.Nullary` is fairly thin. But I guess this could be more future-proof wrt additions to `Relation.Nullary`. Sure, why not.